alf/algorithms/distributed_off_policy_algorithm.py

```diff
@@ -358,6 +358,13 @@ def __init__(self,
         # may be incremented every mini-batch
         self._num_train_iters = 0
 
+        # respect core_alg's replay buffer setting
+        self._num_earliest_frames_ignored = self._core_alg._num_earliest_frames_ignored
+
+        # We always test tensor sharing among processes, because
+        # we rely on undocumented features of PyTorch
```

Comment:
Explain what the undocumented feature is.

Reply:
> Explain what the undocumented feature is.

added explanation